### ISSUE TITLE
[Windows] If you install as administrator and launch from setup, owncloud.exe is owned by the admin

### ISSUE THREAD

**Issue by jasonblewis** [owncloud/client#5608]
### Expected behaviour
installing ownCloud should be possible from a non privileged account by using the Run As Administrator. after running the install, it should then prompt to run the owncloud client as the current user, not as the administrator.
### Actual behaviour
it runs the client as the administrator, so when you do the install, it wants to set the home folder to C:\users\Administrator\ownCloud rather than the user's home directory

### Steps to reproduce
1. download owncloud client as a non privileged user
2. run the install by using the Run As Administrator feature of windows 
3. after the install, click yes and ensure "Run ownCloud" is ticked
4. click finish
5. in the owncloud wizard enter server, login and pw
6. on the Setup Folder Locations page of the wizard, note that it suggests C:\Users\Administrator\ownCloud rather than C:\Users\<username>\ownCloud

### Server configuration
Operating system:
linux
Web server: 
apache
Database:
N/A
PHP version:
N/A
ownCloud version:
N/A
Storage backend (external storage):

### Client configuration
Client version:
2.3.0.6780
Operating system:
Windows Server 2016 Standard
OS language:
English
Qt version used by client package (Linux only, see also Settings dialog):

Client package (From ownCloud or distro) (Linux only):

Installation path of client:

### Logs

Please use Gist (https://gist.github.com/) or a similar code paster for longer
logs.

```Template for output < 10 lines```

1. Client logfile: Output of `owncloud --logwindow` or `owncloud --logfile log.txt`
(On Windows using `cmd.exe`, you might need to first `cd` into the ownCloud directory)
(See also http://doc.owncloud.org/desktop/2.2/troubleshooting.html#client-logfile )

2. Web server error log:

3. Server logfile: ownCloud log (data/owncloud.log):

**Comment by hodyroff** [comment]
This has always been the case, or? (= no regression).
@SamuAlfageme please reproduce.

**Comment by SamuAlfageme** [comment]
@jasonblewis

> after running the install, it should then prompt to run the owncloud client as the current user, not as the administrator

Logged as a normal (non-privileged user) I'm not being able to reproduce this with v.2.3.0 of the client; could you verify what `User name` appears in the task manager (`ctrl+alt+del`) after running the installer?

![screenshot 2017-03-14 09 21 32](https://cloud.githubusercontent.com/assets/2644445/23891646/a750b524-0897-11e7-9ec2-ba5d20f5d471.png)

Also the config is stored in the right [`%LOCALAPPDATA%\ownCloud\owncloud.cfg` file](https://doc.owncloud.org/desktop/2.2/advancedusage.html#configuration-file): `%LOCALAPPDATA%` expands to `C:\Users\samuel\AppData` in this case. And the same goes for the reproduction step no.6. 

However, I tried to reproduce with Windows 10. Might have something to do with Windows Server 2016 specifics... 🤔

**Comment by guruz** [comment]
@SamuAlfageme And you ran it from the setup wizard?

@dschmidt Maybe it's an idea to just disable/hide that [X] Run checkbox when NSIS is ran as installer

**Comment by SamuAlfageme** [comment]
@guruz yes, right from the installer and the process was owned by the user, not the admin.

**Comment by jasonblewis** [comment]
I uninstalled and re-installed (from the same downloaded installer). This time I didn't use the Run As, and the installer did what I expected. ie, installed fine and ran as the correct user.

previously I used the Run As feature as I'm sure the other way gave me an error but this time it didn't.

I guess as long as you don't use the Run As feature, its ok? let me know if you want me to test that again?

**Comment by guruz** [comment]
I still think we could improve this..

**Comment by jasonblewis** [comment]
I ran through the Run As procedure again, and got the same result. this time I installed using ownCloud-2.3.1.6812rc1-setup.exe

see attached screenshots. it runs the owncloud client as Administrator. not really surprising as how would the owncloud installer know it was run by the "run as" command as Administrator? 

![capture](https://cloud.githubusercontent.com/assets/129165/24026841/ec80c832-0b17-11e7-8605-a500b2d08457.PNG)

note that its suggesting to install in C:\Users\Administrator\ownCloud, but the currently logged in user is jason

![capture2](https://cloud.githubusercontent.com/assets/129165/24026852/f5638e6c-0b17-11e7-928a-148ff629c0fc.PNG)

hth

**Comment by phil-davis** [comment]
I tried the Windows installers of some other software, using "Run as Administrator" to start them:
1) IrfanView - if I select to "Start IrfanView" after the install is finished, then the application starts as the admin user that I ran the install with.
2) VLC Media Player - if I select to "Stat VLC" after the install is finished, then the application starts as the (unpriv) user that I logged in t0 the desktop.

(1) is what I expected to happen, because the installer is running in a process owned by some admin user name, and would not really know what unpriv user name to use to start the application.

But (2) indicates that the VLC installer has some smarts to work out what to do. Maybe the admin process that the installer is running in "knows" somehow where it came from (what unpriv user name invoked it), and so the VLC installer can find that out and thus, when the install is finished, start the app process "guessing" the desired user name.

The easy workaround is that when installing, the user can just run the installer and let Windows detect that it needs priv, and so Windows will make the switch to admin and "switch back" at the end...

But it does seem that it would be possible to do whatever VLC does.
